index.js

```diff
@@ -201,7 +201,12 @@ Client.prototype.getPayment = function(hash, callback) {
       return callback(error);
     }
 
-    callback(null, response.body.payment);
+    var payment = response.body.payment;
+    payment.hash = response.body.hash;
```

Comment:
Would be cleaner to clone and modify, since you're modifying a reference to the payment object that's part of the body that is part of the request.